config/initializers/okcomputer.rb

```diff
@@ -59,6 +60,16 @@ def check
 
 OkComputer::Registry.register 'ruby_version', OkComputer::RubyVersionCheck.new
 
+# check for backed up resque queues
+Resque.queues.each do |queue|
+  OkComputer::Registry.register "feature-backed-up-#{queue}",
+                                OkComputer::ResqueBackedUpCheck.new(queue, 5_000_000)
+end
+
+# check for failed resque jobs
```

Comment:
Can we / should we do this per error queue instead?

If we keep this, what do you think of as "feature-resque-failures-threshold" as the name?

Reply:
The native OKComputer Resque tools don't have the ability to monitor per error queue, just the total number of errors. 

I'm willing to look into writing custom checks, if we think there's enough of a need.

+1 to the name.